[AOTInductor] Do not hardcode directory with .cubin files

khabinov opened the pull request:
Reviewed By: frank-wei, chenyang78

Differential Revision: D49081883




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

chenyang78 commented:
Thanks, @khabinov! LGTM overall. You may need to sign the CLA. There are some linter issues that can be addressed by running from the OSS side (you can checkout the pull request locally)

```
$ lintrunner -a
```

Also, some CI failure seems to be legit:

https://github.com/pytorch/pytorch/actions/runs/6166139422/job/16735810926

Could you take a look? Thanks!

chenyang78 reviewed (approved):
LGTM. Thanks!

khabinov commented:
@pytorchbot label "topic: not user facing"

khabinov commented:
@pytorchbot merge

khabinov committed:
[AOTInductor] Do not hardcode directory with .cubin files (#109151)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/109151

Reviewed By: frank-wei, chenyang78

Differential Revision: D49081883
